05-onchain.md

```diff
@@ -424,74 +361,68 @@ A node MUST resolve all unresolved outputs as follows:
    already known.
 
-
 The node MAY use a single transaction to *resolve* all the outputs, but MUST handle its transactions being invalidated by HTLC transactions.
 
-
 ## Rationale
 
-
-A single transaction which resolves all the outputs will be under the
+A single transaction that resolves all the outputs will be under the
 standard size limit thanks to the 483 HTLC-per-party limit (see
 [BOLT #2](02-peer-protocol.md#the-open_channel-message)).
 
-Note that if a single transaction is used, it may be invalidated as B
+Note that if a single transaction is used it may be invalidated as node B
 broadcasts HTLC-timeout and HTLC-success transactions, but the
-requirement that we persist until all outputs are irrevocably resolved
+requirement of persistence until all outputs are irrevocably resolved
 should cover this. [FIXME: May have to divide and conquer here, since they may be able to delay us long enough to avoid successful penalty spend? ]
 
 ## Penalty Transactions Weight Calculation
 
-There are three different scripts for penalty transactions, with the following witnesses weight (details of the computation in [Appendix A](#appendix-a-expected-weights)):
+There are three different scripts for penalty transactions, with the following witnesses weight (details of the computation are in [Appendix A](#appendix-a-expected-weights)):
 
     to_local_penalty_witness: 160 bytes
     offered_htlc_penalty_witness: 243 bytes
     accepted_htlc_penalty_witness: 249 bytes
 
-The penalty txinput itself takes 41 bytes, thus has a weight of 164, making the weight of each input:
+The penalty txinput itself takes 41 bytes, which has a weight of 164, making the weight of each input:
 
     to_local_penalty_input_weight: 324 bytes
     offered_htlc_penalty_input_weight: 407 bytes
     accepted_htlc_penalty_input_weight: 413 bytes
 
-
 The rest of the penalty transaction takes 4+1+1+8+1+34+4=53 bytes of non-witness
-data, assuming it has a pay-to-witness-script-hash (the largest standard output
-script), in addition to a 2 byte witness header.
+data, assuming that it has a pay-to-witness-script-hash (the largest standard output
+script), in addition to a 2-byte witness header.
 
 In addition to outputs being swept under as penalty, the node MAY also sweep the
-`to_remote` output of the commitment transaction, e.g. to reduce the total
-amount paid in fees. Doing so requires the inclusion of a p2wpkh witness and
+`to_remote` output of the commitment transaction (e.g. to reduce the total
+amount paid in fees). Doing so requires the inclusion of a p2wpkh witness and
 additional txinput, resulting in an additional 108 + 164 = 272 bytes.
 
-In a worst case scenario, we have only incoming HTLCs and the HTLC-timeout
-transactions are not published, forcing us to spend from the commitment
+In a worst case scenario, a node holds only incoming HTLCs and the HTLC-timeout
+transactions are not published, forcing them to spend from the commitment
```

Comment:
...HTLCs, and...published, which forces the node to...

Reply:
Changed.